schema/Cargo.toml

```
@@ -7,8 +7,8 @@ edition = "2021"
 [dependencies]
 iroha_schema_derive = { version = "=2.0.0-pre.1", path = "derive" }
 
-serde = { version = "1.0", features = ["derive"] }
-fixnum = { version = "0.6", features = ["i64"]}
+serde  = { version = "1.0.132", default-features = false, features = ["derive", "alloc"] }
```

```suggestion
serde = { version = "1.0.132", default-features = false, features = ["derive", "alloc"] }
```
This is a very minor æsthetic point, but consistent single space would be very much easier to work with.